npm/npm_import.bzl

```diff
@@ -325,6 +325,7 @@ def npm_translate_lock(
     additional_file_contents = kwargs.pop("additional_file_contents", {})
     repositories_bzl_filename = kwargs.pop("repositories_bzl_filename", None)
     defs_bzl_filename = kwargs.pop("defs_bzl_filename", None)
+    generate_bzl_library_targets = kwargs.pop("generate_bzl_library_targets", None)
```

Comment:
`False` instead of `None` since `generate_bzl_library_targets` is `attr.bool()`?

Reply:
`None` implies the default gets used which is better so the default doesn't have to be encoded in two places

Reply:
Isn't the second param of `pop` the value returned if the first param is not in the map?

Reply:
Or you mean passing `None` from here into the actual rule means it will use the rule default?